sdk/python/core/python.cpp

```diff
@@ -485,8 +485,7 @@ PYBIND11_PLUGIN(ydk_)
     class_<ydk::NetconfServiceProvider, ydk::path::ServiceProvider>(providers, "NetconfServiceProvider")
         .def("__init__",
             [](ydk::NetconfServiceProvider &nc_provider, ydk::path::Repository& repo, string address, string username, string password, int port, string protocol) {
```

Comment:
Can these variables be changed here and in `netconf_provider.hpp` from std::string to const std::string & to avoid copying?